charts/perses/values.yaml

```diff
@@ -182,13 +183,15 @@ ingress:
 # -- Configure datasources
 # ref: https://github.com/perses/perses/blob/90beed356243208f14cf2249bebb6f6222cb77ae/docs/datasource.md
 datasources:
-  []
-  # - kind: GlobalDatasource
-  #   metadata:
-  #     name: PrometheusDemoBrowser
-  #   spec:
-  #     default: true
-  #     plugin:
-  #       kind: PrometheusDatasource
-  #       spec:
-  #         directUrl: https://prometheus.demo.do.prometheus.io
\ No newline at end of file
+  - kind: Datasource
+    metadata:
+      name: PrometheusLocalhost
+      project: perses
+    spec:
+      default: true
+      plugin:
+        kind: PrometheusDatasource
+        spec:
+          directUrl: http://localhost:9090
+          scrapeInterval: "15s"
+ 
```

Comment:
This here is also a good one for feedback, to make it work out without having the user worry too much, I created a data source by default. If you prefer to keep that as a comment here, no objections, and just let me know what you prefer here.

Reply:
yeah I would prefer to keep it commented. In production environment, it would be annoying to have this datasource.

Probably to make it work easily for user, an idea could be to see prometheus-operator deploying Perses, like it deploys Grafana.